spec.html

```diff
@@ -40671,10 +40671,10 @@ <h1>AsyncFunction.prototype</h1>
       <h1>Properties of the AsyncFunction Prototype Object</h1>
       <p>The AsyncFunction prototype object:</p>
       <ul>
+        <li>is the intrinsic object <dfn>%AsyncFunctionPrototype%</dfn>.</li>
+        <li>is the value of the *"prototype"* property of %AsyncFunction%.</li>
```

Comment:
```suggestion
        <li>is the initial value of the *"prototype"* property of %AsyncFunction%.</li>
```

Reply:
another alternative:
```suggestion
        <li>is %AsyncFunction.prototype%.</li>
```
which implicitly conveys "initial" as well.